test-project/hello.py

```diff
@@ -23,3 +23,5 @@
 plt.show()
 
 print('Open test-project/plot.png to see the result!')
+
+# %%
```

Comment:
Does this comment have any special purpose?

Reply:
haha, no is just a little typo :D